strapi/strapi
# Add <latest> Upgrade Command

### What does it do?

Introduces a new `latest` release type and command for version upgrades.

When using:

`upgrade latest`

The upgrade tool will try to upgrade to the latest available version of Strapi (no other rule applies unlike `major`, `minor`, and `patch`)

### Why is it needed?

Provide an option for users to easily upgrade to the latest available version of Strapi without having to specify a specific release type.

It simplifies the upgrade process, making it more intuitive and flexible, thus hopefully improving the overall developer experience.

### How to test it?

Set up a Strapi environment and use the 'upgrade' command with the new 'latest' command.

Verify that the system correctly chooses and upgrades to the latest version.
